### PR TITLE
Add juliamono w/ git auto-update

### PR DESCRIPTION
**Pull request by cormullion**
Add juliamono font

### CODE REVIEW

File: packages/j/juliamono.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "juliamono-font",
+  "description": "A monospaced font with reasonable Unicode support",
+  "filename": "juliamono.css",
```
Comment (MattIPv4) [inline packages/j/juliamono.json:4]:
```suggestion
```
It looks like you're only distributing the font files, so you can probably just omit the default filename?

**Review by MattIPv4 — CHANGES_REQUESTED** [review]

File: packages/j/juliamono.json (hunk 1 above)
Comment (cormullion) [reply packages/j/juliamono.json]: Thanks - I wasn't quite sure about this... :)

**Review by MattIPv4 — APPROVED** [review]
lgtm, ty!

**Comment by cormullion** [comment]
Thanks Matt!

**Pull request merged**